SPECS/nginx/nginx.spec

```diff
@@ -1,12 +1,13 @@
 %global nginx_user nginx
 %global njs_version 0.7.12
+%global opentelemetry_cpp_contrib_git_commit 37e4466d882cbddff6f607a20fe327060de76166
```

Comment:
Curious how this git commit was chosen. Does it correspond to a specific tag/release?

Reply:
This is the latest `master`.
The project does not follow a good release name pattern unfortunately.